app/src/main/java/com/pvp/app/service/ExerciseServiceImpl.kt

```diff
@@ -28,4 +54,14 @@ class ExerciseServiceImpl @Inject constructor(
             )
         )
     }
+
+    override suspend fun getInfrequentActivities(maxOccurrence: Int): List<SportActivity> {
+        val activityOccurrences = getActivitiesWithOccurrencesMap()
+
+        return SportActivity.entries.filter { activity ->
+            (activityOccurrences[activity] == null
+                    || activityOccurrences[activity]!! <= maxOccurrence)
```

Comment:
All operators in all cases could instead be within the initial line and then split upon combined entry. Just like lambdas.
```kotlin
           (activityOccurrences[activity] == null ||
               activityOccurrences[activity]!! <= maxOccurrence) &&
           activity != SportActivity.Other
// or...
           (
               activityOccurrences[activity] == null ||
               activityOccurrences[activity]!! <= maxOccurrence // this may have to be indented by 4 spaces in kotlin, not sure (Control+Alt+L)
           ) &&
           activity != SportActivity.Other
```